### PR TITLE
Change error message for invalid product quantity

### LINKED ISSUE #18106: [Order page] Improve error message in partial refund
Currently in the partial refund block, if the user does not fill the quantity field but only the price field, he gets the following error message : "Please enter a positive quantity." This leads to misunderstanding. We should modify this error to tell the user to add a quantity in the quantity field.

What do you think @LouiseBonnard ?

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Change error message or invalid product quantity
| Type?         | bug fix
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #18106
| How to test?  | Try to refund a product and set a negative quantity the error message should be `Positive product quantity is required`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18420)

### CODE REVIEW

**Review by sowbiba — DISMISSED** [review]

File: app/Resources/translations/default/AdminOrderscustomersNotification.xlf [hunk 1]
```diff
@@ -929,11 +929,6 @@
         <target>Order #%d cannot be loaded.</target>
         <note>Line: 1408</note>
       </trans-unit>
-      <trans-unit id="3354cb02aba47cf9f44d703b25a0a777">
```
Comment (matks) [inline app/Resources/translations/default/AdminOrderscustomersNotification.xlf:932]: Shouldn't we keep this ? Do BC breaks apply to translations ? What if a module uses this translation ?

Comment (jolelievre) [reply app/Resources/translations/default/AdminOrderscustomersNotification.xlf:932]: No don't worry this wording has been introduced in 177 only, so we can remove it
There are a ew others that need to be clean as well in another issue (currency's wording)

**Review by matthieu-rolland — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @jolelievre . It's almost OK : There is just a missing "." at the end of the message (all messages in PS finish with period).

Thanks !

File: src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php [hunk 2]
```diff
@@ -1430,8 +1430,8 @@ private function getErrorMessages(Exception $e)
             ),
             InvalidCancelProductException::class => [
                 InvalidCancelProductException::INVALID_QUANTITY => $this->trans(
-                    'Please enter a positive quantity.',
-                    'Admin.Orderscustomers.Notification'
+                    'Positive product quantity is required',
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php]:
```suggestion
                    'Positive product quantity is required.',
```

File: src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php [hunk 3]
```diff
@@ -1478,8 +1478,8 @@ private function getErrorMessages(Exception $e)
                 ),
             ],
             InvalidProductQuantityException::class => $this->trans(
-                'Please enter a positive quantity.',
-                'Admin.Orderscustomers.Notification'
+                'Positive product quantity is required',
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php]:
```suggestion
                'Positive product quantity is required.',
```

**Review by Progi1984 — APPROVED** [review]

**Review by LouiseBonnard — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Thanks @jolelievre !

**Comment by matks** [comment]
Thank you @jolelievre

**Pull request merged**